backend/Application/Api/GraphQL/Import/BakerImportHandler.cs

```diff
@@ -188,39 +185,38 @@ or ConcordiumSdk.NodeApi.Types.BakerSetFinalizationRewardCommission
             or ConcordiumSdk.NodeApi.Types.BakerSetBakingRewardCommission);
 
         await UpdateBakersFromTransactionEvents(txEvents, pendingChangeStrategy, importState, resultBuilder);
+
+        // This should happen after the bakers from current block has been added to the database
```

Comment:
```suggestion
        // This should happen after the bakers from current block have been added to the database
```

Reply:
So this is the key change as I understand it, to move the `UpdateCurrentPaydayStatusOnAllBakers` after the internal state has been updated.

Reply:
yes